### DISCUSSION TITLE
Social Media Button For Contact

### DISCUSSION THREAD

**Discussion by RR-Sahoo** [sButtons/sbuttons#1402] [Ideas]
Can we make some social media button groups that are used in websites for contacting like a button groups of LinkedIn, GitHub, Instagram, Twitter, Facebook, YouTube etc. With some amazing animations.

**Comment by shahednasser** [comment]
Hi @RR-Sahoo thank you for suggesting a new idea. Will these buttons be any different than the current Social buttons?

**Comment by RR-Sahoo** [comment]
Yes, the buttons in this repository are single sign in button but I told about the button groups that generally present at the footer and its contain the social media links in their respective icons.

**Reply by shahednasser** [reply]
Thank you for your suggestion @RR-Sahoo but I don't think these buttons are necessary at the moment. Users can use the icon buttons for that.
